chat-plugins/hangman.js

```diff
@@ -130,33 +147,35 @@ class Hangman extends Rooms.RoomGame {
 			result = 2;
 		}
 
-		let output = '<div class="broadcast-' + (result === 1 ? 'red' : (result === 2 ? 'green' : 'blue')) + '">';
-		output += '<p style="text-align:left;font-weight:bold;font-size:10pt;margin:5px 0 0 15px">' + (result === 1 ? 'Too bad! The mon has been hanged.' : (result === 2 ? 'The word has been guessed. Congratulations!' : 'Hangman')) + '</p>';
-		output += '<table><tr><td style="text-align:center;">' + this.hangingMan() + '</td><td style="text-align:center;width:100%;word-wrap:break-word">';
+		const color = result === 1 ? 'red' : (result === 2 ? 'green' : 'blue');
+		const message = result === 1 ? 'Too bad! The mon has been hanged.' : (result === 2 ? 'The word has been guessed. Congratulations!' : 'Hangman');
+		let output = `<div class="broadcast-${color}">`;
+		output += `<p style="text-align:left;font-weight:bold;font-size:10pt;margin:5px 0 0 15px">${message}</p>`;
+		output += `<table><tr><td style="text-align:center;">${this.hangingMan}</td><td style="text-align:center;width:100%;word-wrap:break-word">`;
 
 		let wordString = this.wordSoFar.join('');
 		if (result === 1) {
 			let word = this.word;
 			wordString = wordString.replace(/_+/g, (match, offset) =>
-				'<font color="#7af87a">' + word.substr(offset, match.length) + '</font>'
+				`<font color="#7af87a">${word.substr(offset, match.length)}</font>`
 			);
 		}
 
-		if (this.hint) output += '<div>(Hint: ' + Chat.escapeHTML(this.hint) + ')</div>';
-		output += '<p style="font-weight:bold;font-size:12pt;letter-spacing:3pt">' + wordString + '</p>';
+		if (this.hint) output += `<div>(Hint: ${Chat.escapeHTML(this.hint)})</div>`;
```

Comment:
This should be ``Chat.html`word ${stuff}` `` instead of `` `word ${Chat.escapeHTML(stuff)}` ``